### DISCUSSION TITLE
MAUI question for a Handler not found error

### DISCUSSION THREAD

**Discussion by ACHildebrand** [mono/SkiaSharp#1882] [Q&A]
I'm currently hacking a Xamarin tutorial for multipoint crop (Found Here: https://github.com/HeikkiDev/XFManualCropControl) and using the MAUI libraries I've run across an issue that I can't see the obvious problem causing this.

On run, the application compiles and runs fine until I attempt to load the page in question, when I load that page it crashes with a "Handler not found for view SkiaSharp.Views.Maui.Controls.SKCanvasView"

The only reference in both projects to SKCanvasView is in the XAML (<skia:SKCanvasView x:Name="CropImageCanvas" BackgroundColor="White"/>) so any insight as to why this might be happening, or what this error means, would be appreciated

**Comment by g-sach** [comment]
Hi @ACHildebrand , was it resolved? I also faced same issue today. It seems we need to add a line ".UseSkiaSharp(true)" in MauiProgram.cs class. I'm not getting this error now.

```cs
using SkiaSharp.Views.Maui.Controls.Hosting;

var builder = MauiApp.CreateBuilder();

builder
    .UseSkiaSharp(true)
    .UseMauiApp<App>()`
```

**Reply by mattleibow** [reply]
This is the correct way as of .NET 6.0. We are working on a way to not require this in the startup: https://github.com/dotnet/maui/issues/7258

**Reply by g-sach** [reply]
Thank you for sharing link & fixing code formatting @mattleibow. I fixed the issue in my code and created github account so that I can respond to this question :). So its my first answer, way to go..!!

**Reply by mattleibow** [reply]
Welcome to SkiaSharp discussions! Glad to have you here helping out the community. The formatting for the code is basically 3 backticks and then the language name, like `cs` or `js` or `xaml` depending on the highlighting you want.
